bliss/predict.py

```diff
@@ -23,19 +123,45 @@ def predict(cfg, image, background):
         pred = encoder.encode_batch(batch)
         est_cat = encoder.variational_mode(pred)
 
-    print("{} light sources detected".format(est_cat.n_sources.item()))
+    if (cfg.predict.plot.ifPlot) & (true_plocs is not None):
+        ttc = cfg.encoder.tiles_to_crop
+        ts = cfg.encoder.tile_slen
+        ptc = ttc * ts
+        cropped_image = image[0, 0, ptc:-ptc, ptc:-ptc]
+        cropped_background = background[0, 0, ptc:-ptc, ptc:-ptc]
+        plot_predict(cfg, cropped_image, cropped_background, true_plocs, est_cat)
 
     return est_cat
 
 
-def predict_sdss(cfg):
+def decal_plocs_from_sdss(cfg):
```

Comment:
Would you add docstring to these functions explaining what they do? This one is a bit mysterious